packer/ui.go

```diff
@@ -342,3 +350,29 @@ func (u *MachineReadableUi) Machine(category string, args ...string) {
 func (u *MachineReadableUi) ProgressBar() ProgressBar {
 	return new(NoopProgressBar)
 }
+
```

Comment:
Please define the `TimestampedUi` struct around here so that it's grouped with it's functions.